components/WhyMeditate/AudioWithSubs.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import subs from './subs.json';
+import styles from './AudioWithSubs.module.css';
+import cx from 'classnames';
+
+function findSegments(ms) {
+  const segments = subs.filter(
+    (s) => s.startMs > 0 && s.startMs < ms && s.startMs + s.duration > ms
+  );
+  return segments.map((segment) => {
+    const segmentStart = segment.startMs;
+    const wordOffset = ms - segmentStart;
+
+    return {
+      parts: segment.parts
+        ?.filter((s) => (s.delay || 0) <= wordOffset)
+        .map((part) => part.phrase),
+      key: segment.key,
+    };
+  });
+}
+
+export function AudioWithSubs({ time }) {
```

Comment:
This should be called `Captions` now.
```suggestion
export function Captions({ time }) {
```